sdk/cognitivelanguage/ai-language-text/review/ai-language-text.api.md

```diff
@@ -629,6 +762,36 @@ export enum KnownPiiEntityDomain {
     Phi = "phi"
 }
 
+// @public
+export enum KnownRelationType {
```

Comment:
Is this a new enum? I thought this was from previous versions. Did the name change?

Reply:
I forgot to expose it earlier, but it doesn't affect the functionality of the library. It is just an extra enum to help customers write their applications.